README.md

```diff
@@ -36,6 +36,10 @@ E.g. use of `build.sh` to build the coupled model with `FV3_GFS_v15p2` as the CC
 ```
 $> CMAKE_FLAGS="-DS2S=ON" CCPP_SUITES="FV3_GFS_v15p2" ./build.sh
```

Comment:
I think we need to update the `README.md` to replace `-DS2S=ON` with `-DAPP=S2S`